applications/IgaApplication/IgaApplication.py

```diff
@@ -1,14 +1,12 @@
-#
-#   KRATOS .___  ________    _____
-#          |   |/  _____/   /  _  \
-#          |   /   \  ___  /  /_\  \
-#          |   \    \_\  \/    |    \
-#          |___|\______  /\____|__  /
-#                      \/         \/  Application
-#
-#   License: BSD License
-#            Kratos default license: kratos/license.txt
-#
+# 
+#   KRATOS  _____________
```

Comment:
Sorry but this style is exclusively reserved by the MappingApplication ;)

I will take a look later